config/initializers/sidekiq.rb

```diff
@@ -25,6 +25,8 @@
   redis_url = "rediss://:#{ENV.fetch('REDIS_PASSWORD',
                                      nil)}@#{ENV.fetch('REDIS_HOST',
                                                        nil)}:6379"
+elsif ENV['REDIS_HOST'].present? && ENV['CI'] # for running on CI as no password
+  redis_url = "rediss://#{ENV.fetch('REDIS_HOST', nil)}:6379"
 end
```

Comment:
(This code is taken from CCCD)

we can use an environment variable called INLINE_SIDEKIQ to not have to run in separate process in development. Can add this: 
```
if ENV['INLINE_SIDEKIQ'].eql?('true')
  raise 'Sidekiq must be run using redis in production' unless Rails.env.development?
  require 'sidekiq/testing'
  Sidekiq::Testing.inline!
end
```

and have INLINE_SIDEKIQ=true in the local environment variables

Reply:
OR just have the conditional statement for this be if Rails.env.development?

Reply:
Also, there's an extra s here in redis on line 29

Reply:
@ivanELEC  So we alreadu have Sidekiq set to inline, however one of the test is failed as it tries to connect to redis (see linked PR) I have not idea why this is happening.. and debugging it would be a length of time... adding the redis server makes the tests pass and can be done quickly....

rediss is basically https